### PR TITLE
scripts/utils/migrate_mcumgr_kconfigs.py: Fix missing Kconfigs

### LINKED ISSUE #53306: scripts: utils: migrate_mcumgr_kconfigs.py: Missing options
It seems that this file might be missing options, e.g. `CONFIG_MCUMGR_GRP_OS_OS_RESET_HOOK` to `CONFIG_MCUMGR_GRP_OS_RESET_HOOK` seems to be missing but was applied here: https://github.com/zephyrproject-rtos/zephyr/commit/a5488d64d4d65709613a0ca7a4564e9a8e530285#diff-834286adf361cd54c1b8c09197543580009a48bd6988bc78240a2487ba731f0e

### PR DESCRIPTION
**Pull request by de-nordic**
Add missing Kconfig translations.

Fixes #53306

Signed-off-by: Dominik Ermel <dominik.ermel@nordicsemi.no>

### CODE REVIEW

File: scripts/utils/migrate_mcumgr_kconfigs.py [hunk 1]
```diff
@@ -58,6 +58,7 @@
     "IMG_MGMT_REJECT_DIRECT_XIP_MISMATCHED_SLOT" : "MCUMGR_GRP_IMG_REJECT_DIRECT_XIP_MISMATCHED_SLOT",
     "IMG_MGMT_FRUGAL_LIST" : "MCUMGR_GRP_IMG_FRUGAL_LIST",
     "MCUMGR_CMD_OS_MGMT" : "MCUMGR_GRP_OS",
+    "MCUMGR_GRP_OS_OS_RESET_HOOK", "MCUMGR_GRP_OS_RESET_HOOK",
```
Comment (nordicjm) [inline scripts/utils/migrate_mcumgr_kconfigs.py:61]: : instead of ,

File: scripts/utils/migrate_mcumgr_kconfigs.py [hunk 2]
```diff
@@ -76,6 +77,7 @@
     "MGMT_VERBOSE_ERR_RESPONSE" : "MCUMGR_SMP_VERBOSE_ERR_RESPONSE",
     "MCUMGR_SMP_REASSEMBLY_BT" : "MCUMGR_TRANSPORT_BT_REASSEMBLY",
     "MCUMGR_SMP_REASSEMBLY" : "MCUMGR_TRANSPORT_REASSEMBLY",
+    "MCUMGR_SMP_REASSEMBLY_UNIT_TESTS", "MCUMGR_TRANSPORT_REASSEMBLY_UNIT_TESTS",
```
Comment (nordicjm) [inline scripts/utils/migrate_mcumgr_kconfigs.py:80]: As above

**Review by nordicjm — CHANGES_REQUESTED** [review]

**Comment by de-nordic** [comment]
Fun fact: original list has been generated from script running on diff of Kconfig changes, and somehow two of Kconfigs got missed. Probably I have removed them, accidentally,  when moving some of the Kconfig options around.

**Review by nordicjm — DISMISSED** [review]

**Comment by nordicjm** [comment]
Hmm odd, `CONFIG_MCUMGR_GRP_FS_FILE_ACCESS_HOOK` does not seem to be in this script

**Comment by de-nordic** [comment]
> Hmm odd, `CONFIG_MCUMGR_GRP_FS_FILE_ACCESS_HOOK` does not seem to be in this script

It is already correct, isn't it?
The original script has been generated by doing diff between change and main and then filtering out pairs of
```
-config XXX
+config YYY
```
and generating map of the YYY to XXX from generated list.

**Comment by nordicjm** [comment]
See https://github.com/zephyrproject-rtos/zephyr/pull/53558/commits/e2e76c88c49c13c7d6f2727dce52d838d0b1d52b
It was missing the CONFIG_ prefix, but also the name was using the old name, and the old -> new name is not in this script

**Comment by de-nordic** [comment]
> See [e2e76c8](https://github.com/zephyrproject-rtos/zephyr/commit/e2e76c88c49c13c7d6f2727dce52d838d0b1d52b) It was missing the CONFIG_ prefix, but also the name was using the old name, and the old -> new name is not in this script

Ah, I know what has happened. The correction was done as part of the new events implementation, not renaming. That is why I have failed to detect it.

**Review by nordicjm — APPROVED** [review]

**Review by carlescufi — APPROVED** [review]

**Pull request merged**